src/popup/router/components/Modals/TransferReceive.vue

```diff
@@ -0,0 +1,353 @@
+<template>
+  <Modal class="transfer-receive-modal">
+    <div
+      class="transfer-receive"
+      data-cy="top-up-container"
+    >
+      <Close class="close" />
+
+      <span class="title">{{ $t('modals.receive.title') }}</span>
+
+      <div class="account-info">
+        <Avatar
+          class="avatar"
+          :address="accounts[idx].address"
+          :name="accounts[idx].name"
+        />
+        <Truncate
+          v-if="accounts[idx].name"
+          :str="accounts[idx].name"
+        />
+        <span
+          v-else
+          data-cy="account-name"
+          class="account-name"
+        >
+          {{ $t('pages.account.heading') }} {{ accountIdx + 1 }}
+        </span>
+      </div>
+
+      <div class="address-info">
+        <div class="qrcode-cover">
+          <qrcode-vue
```

Comment:
I suggest to use PascalCase.

Reply:
For the css class?

Reply:
If qrcode-vue tag then I am using the existing one and I guess the library supplies tag in the same manner

Reply:
Yes, I'm refering to the component name. We are using PascalCase everywhere and here the name also should follow the rule ("QrcodeVue").